### PR TITLE
BZ1865805 - Updating ConfigMap additional resources references

### PR DESCRIPTION
**Pull request by pneedle-rh**
Applies to branch/enterprise-4.3, branch/enterprise-4.4 and branch/enterprise-4.5.

Does not apply to master or branch/enterprise-4.6, following the restructuring of the Monitoring book in those branches.

This relates to https://bugzilla.redhat.com/show_bug.cgi?id=1865805.

The preview is [here](https://bz1865805--ocpdocs.netlify.app/openshift-enterprise/latest/monitoring/monitoring-your-own-services.html#enabling-monitoring-of-your-own-services_monitoring-your-own-services) and [here](https://bz1865805--ocpdocs.netlify.app/openshift-enterprise/latest/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html).

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Thanks!

**Comment by bmcelvee** [comment]
LGTM

**Comment by pneedle-rh** [comment]
@juzhao or @lihongyan1 can you please review this from a QE perspective? Thanks!

File: monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc [hunk 1]
```diff
@@ -18,12 +18,24 @@ This section explains what configuration is supported, shows how to configure th
...
+.Additional resources
+
+* See xref:../../monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-monitoring[Creating a cluster monitoring ConfigMap] to learn how to create the `cluster-monitoring-config` ConfigMap object.
+
 include::modules/monitoring-configurable-monitoring-components.adoc[leveloffset=+1]
 include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../../monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-monitoring[Creating a cluster monitoring ConfigMap] to learn how to create the `cluster-monitoring-config` ConfigMap object.
```
[the remark this answers is not in the document]
Comment (pneedle-rh) [reply monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc:31]: @lihongyan1 that is a great suggestion, thank you! Resolved.

File: monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc [hunk 2]
```diff
@@ -18,12 +18,25 @@ This section explains what configuration is supported, shows how to configure th
 include::modules/monitoring-maintenance-and-support.adoc[leveloffset=+1]
 include::modules/monitoring-creating-cluster-monitoring-configmap.adoc[leveloffset=+1]
 include::modules/monitoring-configuring-the-cluster-monitoring-stack.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../../monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-monitoring[Creating a cluster monitoring ConfigMap] to learn how to create the `cluster-monitoring-config` ConfigMap object.
```
Comment (juzhao) [inline monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc:24]: the link does not link to `Creating a cluster monitoring ConfigMap` directly

Comment (pneedle-rh) [reply monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc:24]: @juzhao thank you for your review! This xref works for me, as do all of the other xrefs in this PR. They each take me directly to the module that is listed in the square parenthesis or the top level page if that is intended.

For example, the link in the preceding line navigates in the preview to [Creating a cluster monitoring ConfigMap](https://bz1865805--ocpdocs.netlify.app/openshift-enterprise/latest/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-monitoring).

**Pull request merged**